src/helpers/helpers.js

```diff
@@ -0,0 +1,32 @@
+const flatten = require('flat')
+const { set, get } = require('lodash')
+
+/**
+ * Parse a Firebase Timestamp object to a UTC string
+ */
+const parseFirebaseDate = timestampObject => {
+  if (timestampObject && timestampObject.toDate && typeof timestampObject.toDate === 'function') {
+    return timestampObject.toDate()
+  }
+  return timestampObject
+}
+
+/**
+ * Iterate over an object and parse all Firebase Timestamp objects to UTC strings
+ */
+exports.parseFirebaseTimestamps = entry => {
+  const flattenedEntry = flatten(entry)
+  /**
+   * After flattening the entry a Firebase timestamp can be found by looking for keys that
+   * contains either a `_seconds` or a `_nanoseconds` property. We'll look for the `_seconds`
+   * key. Once found we know that the related entry is a Firebase timestamp and we pass
+   * it on to be parsed into a UTC string.
+   */
+  Object.keys(flattenedEntry)
+    .filter(key => key.includes('_seconds'))
```

Comment:
This check might cause false positives. Imagine I had a schema field key as `created_date_in_seconds`. It will probably be better to check for `key.endsWith('._seconds')` with the leading dot. What do you think?

Reply:
Yeah, can do that. Luckily above it will also only try conversion if it has the `toDate` method. But rather doubly safe than sorry.

Reply:
Ok, made this change and tested again. Also with the "all fields" JSON file and it all looked good. I tested with a `yolo_seconds` field and with the changes that won't get picked up.